### ISSUE TITLE
[feat]: Ability to host multiple sites

### ISSUE THREAD

**Issue by kn0wmad** [Start9Labs/start9-pages-startos#9]
### Prerequisites

- [X] I have searched for existing issues that already suggest this feature.

### Describe the Feature Request

Self-explanatory.  It may be preferred as an OS option to run multiple instances of any particular service, which is a whole can-o-worms

### Describe the Use Case

Multiple Onion site-hosting from one device

**Comment by elvece** [comment]
You can effectively do this by having different subdomains point to different folders in Filebrowser.

**Comment by kn0wmad** [comment]
Good point, but the idea in specific is to have more than one Hidden Service (onion address).  This can be closed if out of scope for the time being

**Comment by elvece** [comment]
Would have to think about it further and get feasibility insight from @dr-bonez. This service is essentially an nginx server, and I don't think nginx can handle multiple domains in one instance.

**Comment by dr-bonez** [comment]
It can! It's actually what we use it for in most places. The required os feature for this would be to be able to dynamically allocate interfaces from the os at runtime, instead of statically during install

**Comment by elvece** [comment]
> It can! It's actually what we use it for in most places. The required os feature for this would be to be able to dynamically allocate interfaces from the os at runtime, instead of statically during install

@dr-bonez just wondering if this will be made possible by the interfaces refactor?

**Comment by dr-bonez** [comment]
It will
